code/datums/diseases/_disease.dm

```diff
@@ -105,7 +105,7 @@
 		switch(severity)
 			if(DISEASE_SEVERITY_POSITIVE) //good viruses don't go anywhere after hitting max stage - you can try to get rid of them by sleeping earlier
 				cycles_to_beat = max(DISEASE_RECOVERY_SCALING, DISEASE_CYCLES_POSITIVE) //because of the way we later check for recovery_prob, we need to floor this at least equal to the scaling to avoid infinitely getting less likely to cure
-				if((HAS_TRAIT(affected_mob, TRAIT_NOHUNGER)) || affected_mob.nutrition < NUTRITION_LEVEL_STARVING || affected_mob.satiety < 0 || slowdown == 1) //any sort of malnourishment/immunosuppressant opens you to losing a good virus
+				if((HAS_TRAIT(affected_mob, TRAIT_NOHUNGER)) || ((affected_mob.nutrition > NUTRITION_LEVEL_STARVING) && (affected_mob.satiety >= 0)) && slowdown == 1) //any sort of malnourishment/immunosuppressant opens you to losing a good virus
```

Comment:
```suggestion
				if(HAS_TRAIT(affected_mob, TRAIT_NOHUNGER) || ((affected_mob.nutrition > NUTRITION_LEVEL_STARVING) && (affected_mob.satiety >= 0) && (slowdown == 1))) //any sort of malnourishment/immunosuppressant opens you to losing a good virus
```

this if() check makes zero sense in its current state and is confusing as hell to read, is this what you meant? either we have TRAIT_NOHUNGER _or_ we meet all three criteria to being malnourished? if this isn't what you intended, really consider where you're placing those parenthesis.

Reply:
slowdown isn't a criterion for malnourishment, it's spaceacillin/TRAIT_VIRUS_RESIST

what I'm trying to get at and maybe would be better broken out of one line is that 

if we have trait_nohunger, malnutrition/ poor satiety doesn't matter
but if we have trait_nohunger, slowdown being something other than 1 (unmodified) still does

Reply:
![image](https://github.com/tgstation/tgstation/assets/3894717/4ae9cedd-9232-4030-a64a-a053bb517d1a)
if it helps at all, this is what I'm working with in vsc

Reply:
If anything, I'm realizing that's now also wrong for how it interacts with NOHUNGER, and it should probably instead be this - even though what I did fixed the problem with nutrition for regular mobs, I missed how that was interacting with values that aren't really supposed to be there (nutrition/satiety with ethereals for instance.)

![image](https://github.com/tgstation/tgstation/assets/3894717/e0d87568-4f29-4dd8-9f49-5d281085d41b)